README.md

```diff
@@ -66,6 +66,8 @@ Vertical or horizontal lines are supported.
 	// ID of the scale to bind onto
 	scaleID: 'y-axis-0',
 
+	useTicks: true, // optional, and helpful if using 'x-axis-0' with ticks for data and mode = 'vertical'
```

Comment:
We shouldn't add a separate option for this. Let's auto-detect based on the scale configuration instead.